src/stories/4-elements/alert/alert.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { FrontendEngine } from "../../../components";
+import { IAlertSchema } from "../../../components/elements";
+import { CommonFieldStoryProps, ExcludeReactFormHookProps } from "../../common";
+
+export default {
+	title: "Element/Alert",
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title>Alert</Title>
+					<Description>
+						This component renders a `Alert` component provided by the Design System within a Frontend
+						Engine generated form
+					</Description>
+					<Heading>Props</Heading>
+					<Description>
+						Please refer to the [design
+						system](https://designsystem.life.gov.sg/react/index.html?path=/docs/modules-alert--alert) for
+						the properties of `Alert`
+					</Description>
+					<ArgsTable story={PRIMARY_STORY} />
+					<Stories includePrimary={true} title="Example" />
+				</>
+			),
+		},
+	},
+	argTypes: {
+		...ExcludeReactFormHookProps,
+		...CommonFieldStoryProps("alert", true),
```

Comment:
even though we are directing devs to follow the documentation in the design system, we still want to list the props of interest and showcase notable features for easy reference, this also helps prevent disparity in the event of an update on the design system

in this case, we want to showcase the various `type` and the `actionLink`